<launch>
  <!-- Load all PezController params from YAML into the nodeâ€™s private namespace -->
  <rosparam
    file="$(find pez)/config/pez_params.yaml"
    command="load"
    ns="pez"/>

  <!-- Launch the movement controller node (it will pick up params from above) -->
  <node
    pkg="pez"
    type="pez_axis_controller.py"
    name="move_controller"
    output="screen"/>
</launch>
